src/actions/guides/http_and_routing/routing_and_params.cr

```diff
@@ -336,7 +336,99 @@ class Guides::HttpAndRouting::RoutingAndParams < GuideAction
 
     > Learn more about [error handling](#{Guides::HttpAndRouting::ErrorHandling.path}).
 
-    ## Query parameters
+    ## Handling parameters
+
+    Parameters, or `params`, are data that is sent from client back to the server. There are a few different ways this can happen:
+
+    * Path parameters - The dynamic values passed in your route path. e.g. `/users/:id`.
+    * Query parameters - These are added to the end of a URL after the `?` in key/value pairs. e.g. `?page=1`
+    * Form parameters - When you perform an HTTP POST, the data sent through the body. This may be formatted as JSON, or form data
+    * Multipart parameters - Similar to form parameters, but generally to contain a file
+
+    When parameters are passed to an action, you have access to these with the `params` method.
+
+    ```crystal
+    # src/actions/users/index.cr
+    class Users::Index < BrowserAction
+
+      # example.com/users?page=1&filter=active
+      get "/users" do
+        # These are both type `String`
+        filter = params.get(:filter)
+        page = params.get(:page)
+
+        # This is missing and will raise an error
+        per = params.get(:per)
```

Comment:
I think this is fine, but it made me want to try out an alternate format.
```suggestion
        filter = params.get(:filter) # type String
        page = params.get(:page) # type String
        per = params.get(:per) # Error! there is no parameter :per
```

You can pick which one you like best 😄